duckduckgo/macos-browser
# Support close tabs to left and move close actions to submenu

Task/Issue URL: https://app.asana.com/0/1199230911884351/1207117125840765/f
Tech Design URL:
CC: @ayoy 

**Description**:
This PR adds the option to close all tabs to the left when right clicking a tab. It also moves this action as well as "close to right" and "close other tabs" to a submenu in the tab context menu.

**Steps to test this PR**:
1. Open some tabs
2. Right click a tab, ensure "Close Other Tabs" opens a sub menu
3. Clicking "Close Tabs to the Left" should close all tabs to the left of the clicked tab except pinned tabs
4. "Close to the right" and "Close all other tabs" should work as expected.



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
